### PR TITLE
feat(example): env to json

### PR DESCRIPTION
**Pull request by tty47**
hello!

this test adds the content of the `.env` file as json into the container.

cheers! 🚀

### CODE REVIEW

**Review by mojtaba-esk — APPROVED** [review]
Well done.
Just a question, is passing envs as JSON a common practice? I mean how applications read it and how it translate it to ENV vars?

File: pkg/knuu/knuu.go [hunk 1]
```diff
@@ -36,6 +37,11 @@ var (
 
 // Initialize initializes knuu with a unique scope
 func Initialize() error {
+	err := godotenv.Load()
+	if err != nil {
+		return ErrCannotLoadEnv.Wrap(err)
```
Comment (mojtaba-esk) [inline pkg/knuu/knuu.go:42]: [question] 
- Does the `.env` file have to exist for all tests?
- if it does not exist will it through an error here?

File: pkg/knuu/knuu.go [hunk 2]
```diff
@@ -36,6 +37,11 @@ var (
...
+	}
```
Comment (smuu) [inline pkg/knuu/knuu.go:43]: Please move this to `InitializeWithScope()` so it's always loaded.

**Review by smuu — CHANGES_REQUESTED** [review]

File: .env.example [hunk 3]
```diff
@@ -0,0 +1,3 @@
+KNUU_TIMEOUT=60m # The timeout for the tests
+KNUU_BUILDER=docker # The builder to use for building images. docker or kubernetes
+DEBUG_LEVEL=info # The debug level. debug, info, warn, error
```
Comment (smuu) [inline .env.example:3]: I think this is called `LOG_LEVEL`

```suggestion
LOG_LEVEL=info # The debug level. debug, info, warn, error
```

File: .env.example
Comment (smuu) [inline .env.example:1]: Having this `.env` in this repo with good values would be nice when we run the tests.

**Review by smuu — CHANGES_REQUESTED** [review]
Nice one, only small things

**Review by smuu — APPROVED** [review]
Let's go! :rocket:

**Pull request merged**